src/main/java/securbank/dao/CreditCardDao.java

```diff
@@ -0,0 +1,14 @@
+package securbank.dao;
+
+import java.util.List;
+import java.util.UUID;
+
+import securbank.models.CreditCard;
+
+public interface CreditCardDao extends BaseDao<CreditCard, UUID> {
+
+	public List<CreditCard> findAll();
+	public CreditCard findByCcId(UUID id);
+	public CreditCard findByAccountNo(UUID accountID);
```

Comment:
Please change it to AccountNumber and parameter should be `string accountNumber`